packages/@aws-cdk/aws-appmesh/test/test.route.ts

```diff
@@ -4,55 +4,33 @@ import { Test } from 'nodeunit';
 import * as appmesh from '../lib';
 
 export = {
-  'Can export existing route and re-import'(test: Test) {
+  'Can import Routes using ARN and attributes'(test: Test) {
+    const app = new cdk.App();
     // GIVEN
-    const stack = new cdk.Stack();
-
-    // WHEN
-    const mesh = new appmesh.Mesh(stack, 'mesh', {
-      meshName: 'test-mesh',
-    });
-
-    const router = new appmesh.VirtualRouter(stack, 'router', {
-      mesh,
+    const stack = new cdk.Stack(app, 'Imports', {
+      env: { account: '123456789012', region: 'us-east-1' },
     });
+    const meshName = 'test-mesh';
+    const virtualRouterName = 'test-virtual-router';
+    const routeName = 'test-route';
+    const arn = `arn:aws:appmesh:us-east-1:123456789012:mesh/${meshName}/virtualRouter/${virtualRouterName}/gatewayRoute/${routeName}`;
 
-    const service1 = new appmesh.VirtualService(stack, 'service-1', {
```

Comment:
Same comment here - leave the old test as-is, and create a new one for `fromRouteAttributes`.

Reply:
Since these tests are not actually asserting anything, I'm going to opt to not add them back. I did split the new test into two separate ones that test import by ARN and import by attributes separately. Let me know if you disagree with this path.